components/user_settings/notifications/email_notification_setting/index.ts

```diff
@@ -11,10 +11,17 @@ import {savePreferences} from 'mattermost-redux/actions/preferences';
 import {getCurrentUserId} from 'mattermost-redux/selectors/entities/common';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 import {get as getPreference} from 'mattermost-redux/selectors/entities/preferences';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {ActionFunc} from 'mattermost-redux/types/actions';
 
 import EmailNotificationSetting from './email_notification_setting';
 
-function mapStateToProps(state) {
+type Actions = {
+    savePreferences: (currentUserId: string, emailIntervalPreference: {}) =>
```

Comment:
You changed the type of emailIntervalPreference in the other file, but not in this one. Is it a mistake, or there is any reason to leave it like this?

Reply:
That's my mistake, gonna fix it now.

Reply:
That's my mistake, gonna fix it now.